m4/ax_python_devel.m4

```diff
@@ -246,7 +237,7 @@ EOD`
 			PYTHON_LDFLAGS="-L$ac_python_libdir -lpython$ac_python_version"
 		fi
 
-		if test -z "$PYTHON_LDFLAGS"; then
+		if test -z "PYTHON_LDFLAGS"; then
```

Comment:
looks like it's been like that upstream since v1. Clearly our serial 8 is a modified version. What do you propose to do? Modify it back?